tutorial/consuming_packages.rst

```diff
@@ -29,3 +29,4 @@ you can use them to cross-compile your application to different platforms.
    consuming_packages/different_configurations
    consuming_packages/the_flexibility_of_conanfile_py
    consuming_packages/cross_building_with_conan.rst
+   consuming_packages/intro_to_versioning
```

Comment:
I'm missing a brief introduction above, before "After that, ... to different platforms."